### PR TITLE
Task.01 - Create a welcome page for students

### LINKED ISSUE #5: Task.01 - Create a Welcome Page for Students (as well as a common Navbar with two other pages: About and How to Use)
Create a welcome page for student login with the following elements: a welcome screen prompting the student user to login or create an account

### LINKED ISSUE #8: Task.06 - Create a Welcome Page for Teachers
Create a welcome page for teachers to login with the following elements: a welcome screen prompting the user to login or create a new account

### PR DESCRIPTION
**Pull request by JenLys**
- create a welcome page for student login with the following elements: a welcome screen prompting the user to login

### CODE REVIEW

File: client/src/Welcome/Welcome.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+               
+                <!--This will lead the user to either create a Teacher account or a Student Account-->
+                <p class="create-link"><a href="#" class="link">Don't have an account? Create one</a></p>
+
+                
+            </div>
+        </div>
+    </body>
+</html>
+*/
```
Comment (Vinuyans) [inline client/src/Welcome/Welcome.jsx]: remove the comments before merging

File: client/src/Welcome/styles.css [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+   
+    
+
+ /*
+   position: absolute;
+    right: 0;
+    bottom: 0;
+    z-index: -1;
+    */
+
```
Comment (Vinuyans) [inline client/src/Welcome/styles.css]: ditto

**Review by Vinuyans — APPROVED** [review]

**Comment by Vinuyans** [comment]
@Marian-Hristov Could you merge this for us please

**Comment by JenLys** [comment]
NB: Task .06 is linked to Task .01 (both were worked on the same branch titled "Task.01")

**Comment by JenLys** [comment]
Info for the TA: extra functionalities have been added to the Welcome page on top of what was required in the User Stories. Our welcome page provides "How To Use" and "About" pages to improve the user experience on our platform.

**Pull request merged**